### ISSUE TITLE
Parameter settings for mono+stereo training:

### ISSUE THREAD

**Issue by IkeFwd** [nianticlabs/monodepth2#431]
In mono training, the model I trained can reproduce the results of the paper very well,
But in mono+stereo, there is a big gap between the results of the model I trained and the model provided by code, so, in the mono+stereo training, some parameters need to be adjusted, such as: --scheduler_step_size?

Thank you so much for sharing, this is my favorite work on depth estimation.

Below is the result of my trained model
![image](https://user-images.githubusercontent.com/29300743/168623268-4cd6ac17-1fa3-4de0-98d3-1df61db9a62c.png)

Below are the results for your model：
![image](https://user-images.githubusercontent.com/29300743/168624009-b0a5c4c4-f458-4264-94eb-90bfe530cc85.png)
since I was successful in mono, all I thought might be a hyperparameter issue. can you provide some pointers？

**Comment by daniyar-niantic** [comment]
Hi. Are you using pytorch/torchvision version as specified in the readme? We noticed that later versions of pytorch do not reproduce results that are in the paper.

**Comment by IkeFwd** [comment]
Since my gpu is 3090, it doesn't support cuda10, so I used troch1.8.1, cuda11. In order to reproduce the results of the paper, I modified the order of the initial learning rate of the code, and the default parameters of grid_sample(), as follows:
![image](https://user-images.githubusercontent.com/29300743/168629420-c8e36216-8a83-4234-ae46-c3daadfa49be.png)

![捕获3](https://user-images.githubusercontent.com/29300743/168630283-6624768b-9f59-452a-8f15-9088958f392e.PNG)

Now the  results can be reproduced very well on mono and stereo, I thought it should be possible to achieve the same effect on mono+stereo, maybe I should try it on other devices?

**Comment by IkeFwd** [comment]
I'm sorry I just said it wrong,

I just went to check the results, I only reproduced the results on mono, and the results on stereo also have some gaps with the paper, I'm very sorry. Maybe I need to check some more code, thanks a lot.

![image](https://user-images.githubusercontent.com/29300743/168632655-4130dbe0-af3a-450c-a1b5-28b9c72a13b2.png)

**Comment by daniyar-niantic** [comment]
We are not sure what changes in pytorch made this impact, there are many potential causes. See https://github.com/nianticlabs/monodepth2/issues/414

**Comment by Akilax0** [comment]
@bizheyandebing which python version did you use here?
